### PR TITLE
Mobile tab title

### PR DESCRIPTION
**Pull request by chucklorenz**
Truncating tab titles on mobile devices isn't always effective. See images.

2 changes proposed:
- Added a mobileTabTitle to allow the author to explicitly assign the desired text. 
- Also centered the item graphic on mobile.
  (In the images you'll notice we retained the top tab layout in mobile. This is not germane to the PR, but I did want anyone to get distracted by thinking it was.) 

screen shot 2015-10-24 at 4 48 24 pm  

screen shot 2
screen shot 3

### CODE REVIEW

**Comment by moloko** [comment]
@chucklorenz so sorry I've only just seen this. I think this is a great idea - unfortunately you now have some conflicts, could you resolve these and then I will give this a test?

File: js/adapt-tabs.js [hunk 1]
```diff
@@ -51,6 +52,21 @@ define(function(require) {
 			});
 		},
 
+		setTabText: function() {
+			var items = this.model.get('_items');
+			if (Adapt.device.screenSize == 'small') {
```
Comment (moloko) [inline js/adapt-tabs.js]: Can I suggest the following to replace this `if` statement? Not only is it DRYer but it allows for the standard tab title to be used as a fallback on mobile if the mobileTitle hasn't been specified.

```
var items = this.model.get('_items');
_.each(items, function (item, index) {
    var tabTitle = (Adapt.device.screenSize === 'small' && item.mobileTabTitle) ? item.mobileTabTitle : item.tabTitle;
    this.$('.tabs-navigation-item-inner').eq(index).html(tabTitle);
});
```

Comment (chucklorenz) [inline js/adapt-tabs.js]: Absolutely. I’ll update the PR. I Appreciate the help.

**Review by tomgreenfield — CHANGES_REQUESTED** [review]
Looks like this PR is reverting the changes from @moloko's [commit](https://github.com/cgkineo/adapt-tabs/commit/0aa60c64f9a95c384f64f6d90f328123bf4da0b1). Would you mind updating?

**Pull request closed without merging**